cfme/tests/intelligence/reports/test_widgets.py

```diff
@@ -71,7 +71,7 @@ def _finalize():
         with update(dashboard):
             dashboard.widgets = default_widgets
     request.addfinalizer(_finalize)
-    dashboard.reset_widgets()
+    DefaultDashboard.reset_widgets()
```

Comment:
I would have thought we want to ensure we are reseting on the dashboard we expect to?